terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/VariationExceptionLoggingFilter.java

```diff
@@ -0,0 +1,49 @@
+package org.terasoluna.gfw.functionaltest.app.exceptionhandling;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+import org.terasoluna.gfw.web.exception.ExceptionLoggingFilter;
+
+public class VariationExceptionLoggingFilter extends ExceptionLoggingFilter {

```

Comment:
@yoshikawaa
I think that `ExceptionCodeExceptionLoggingFilter` is appropriate because it expresses the contents of class briefly.
Therefore, I modify it to `ExceptionCodeExceptionLoggingFilter`.